src/Stryker.Core/Stryker.Core/Options/TestFailureBreakBehavior.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Stryker.Core.Options
+{
+    public enum TestFailureBreakBehavior
```

Comment:
I don't think we need this many options. Either the user does not want any failing unit tests, or stryker decides what is an acceptable level of test failure where we can still have an accurate result for the non-failing tests.

Reply:
So do I need to change this enum to simple bool?

Reply:
I would prefer that yes